inlong-sort-standalone/sort-standalone-source/src/main/java/org/apache/inlong/sort/standalone/sink/cls/ClsSinkContext.java

```diff
@@ -97,20 +101,24 @@ public void reload() {
                 }
             });
 
-            SortTaskConfig newSortTaskConfig = SortConfigHolder.getTaskConfig(taskName);
-            if (newSortTaskConfig == null || newSortTaskConfig.equals(sortTaskConfig)) {
+            TaskConfig newTaskConfig = SortConfigHolder.getTaskConfig(taskName);
+            SortTaskConfig newSortTaskConfig = SortClusterConfigHolder.getTaskConfig(taskName);
+            if ((newTaskConfig == null || newTaskConfig.equals(taskConfig))
```

Comment:
The processing logic of TaskConfig and SortTaskConfig can be placed in two implementation classes that implement the same Handler interface, which will be clearer and more convenient for subsequently phasing out the old processing implementation classes.

Reply:
The ClsSinkContext is not an interface or abstract class, the two implementations will result in massive file modification. Same as other sinks